src/main/java/utils/Ui.java

```diff
@@ -0,0 +1,136 @@
+package utils;
+
+import tasks.Task;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Scanner;
+
+public class Ui {
+
+    private static final String LINE_PREFIX = "|| ";
```

Comment:
Good use of 'final' Strings. Makes sense now that I think about it; configuring the variables to be immutable as it will never be changed anyway. Good coding practice.